src/ports/owner/component.ts

```diff
@@ -33,19 +33,25 @@ export function createOwnersComponent(options: {
       getOwnersQuery(parsedFilters, false)
     )
 
-    const countData: { nfts: { id: string }[] } = await subgraph.query(
-      getOwnersQuery(parsedFilters, true)
-    )
+    let count = 0
+
+    while (true) {
+      const countData: { nfts: { id: string }[] } = await subgraph.query(
+        getOwnersQuery({ ...parsedFilters, skip: count }, true)
+      )
+      count += countData.nfts.length
+      if (countData.nfts.length < 1000) break
```

Comment:
we've a constant for this value
```suggestion
      if (countData.nfts.length < MAX_RESULTS) break
```